### ISSUE TITLE
All questions on graphics and graphics programs for Stronghold Crusader 1

### ISSUE THREAD

**Issue by Russianstory** [PodeCaradox/Gm1KonverterCrossPlatform#2]
Hi. Very necessary stripes of lives. I will try to redraw them. Kinh if you can already unpacked files. I will redraw from and then post it here so that you pack them.

**Comment by PodeCaradox** [comment]
Hello Russianstory, you can unpack all files by yourself and edit them, just donwload the Programm https://github.com/Gaaammmler/Gm1KonverterCrossPlatform/releases click on assets and download the Converter.zip ,after unzip the file open the exe in the folder. After start change the language in the Options Tag to russian, after that you can add your workfolder and Stronghold GM1 Folder and edit the files , if you dont now how to use the programm feel free to question me. 

Maybe edit me on Discord than i can show you how to use and change language. Gaaammmler#1397

**Comment by Russianstory** [comment]
How to open gm1 files? I opened the program translated into Russian. so what is next? Where to throw the program folder itself - Converter so that it opens gm1 files

**Comment by PodeCaradox** [comment]
After translation in the programm click on the folowing two menue point and select GM1 Folder (in the Stronghold Folder!!!(for me it is C:\Program Files (x86)\Steam\steamapps\common\Stronghold Crusader Extreme\gm) also works with Stronghold 1) and your own Workfolder(for me i did it in my GM1 folder and named it Workfolder:(C:\Program Files (x86)\Steam\steamapps\common\Stronghold Crusader Extreme\gm\Workfolder)):
![img1](https://user-images.githubusercontent.com/5760157/57954882-7e6df700-78f4-11e9-892a-76bdb8174b80.JPG)
After that you can see the GM1 Files on the Right Side:
![img2](https://user-images.githubusercontent.com/5760157/57954883-7e6df700-78f4-11e9-81e2-b08fa1b10790.JPG)

**Comment by Russianstory** [comment]
Listen)) I managed to run !! I even managed to extract the baker's animation footage. Everything is fine for now. This program is much more convenient than the original.

**Comment by Russianstory** [comment]
BUT there is a problem .... is it a color inversion or the colors of units are simply displayed incorrectly and I don’t know how to fix it ....

**Comment by PodeCaradox** [comment]
just click on the arrows next to the colortables and it loads with the corecct colors. And please dont use gifs to edit the images, under the options menue you can export all as png images. There are 10 diffrent Colortables if you load Troups Colortable 1 and 10 are inversions sometimes, you can click on the INFO Icon to read more about the Colortables.


![img1](https://user-images.githubusercontent.com/5760157/57969236-5d99b600-7975-11e9-8179-53124d8f0855.JPG)
![img2](https://user-images.githubusercontent.com/5760157/57969237-5e324c80-7975-11e9-8802-31efdb69e5d3.JPG)
![img3](https://user-images.githubusercontent.com/5760157/57969238-5e324c80-7975-11e9-8287-ba30d6791dd1.JPG)

**Comment by Russianstory** [comment]
Yes, it turned out! you are a professional you made such a program !! When is the release? I would like to post information about her in my group on the social network ... but until you made your official release, I do not dare to do that.

**Comment by PodeCaradox** [comment]
I have the firstbofficial release ready now, the one thing you cant do now is edit animation with colortable but it will comme soon, but all other things working right now. Just look at the releases here and you find out if i have posted a new release
https://github.com/Gaaammmler/Gm1KonverterCrossPlatform/releases

Its better to use my programm right now, besause the img toolbox have some errors i csnt fix right now, please not use GM1 FIlES EDIT in the imagetoolbox, use the orginal Stronghold 1bor crusader Gm1 files.

Of course you can post the link everywhere you want.

**Comment by Russianstory** [comment]
So, I can pour this program on myself and show it to my comrades? To work with graphics? I just sit in several forums. =)

**Comment by PodeCaradox** [comment]
Yea you can post the link for the programm release in several forums :)

**Comment by PodeCaradox** [comment]
I have a new release for import Animations, you can check it out: https://github.com/Gaaammmler/Gm1KonverterCrossPlatform/releases

**Comment by Russianstory** [comment]
it dont works -  E:\Stronghold Crusader Extreme HD\gm\Converter

**Comment by PodeCaradox** [comment]
Can you tell me exactly what not working?

**Comment by Russianstory** [comment]
the program suddenly stopped working. I do not know why. I did not change anything and did not do it. Where should I install it? exact address please Link to the latest version of the program.

**Comment by Russianstory** [comment]
I click on the program crashes.

**Comment by Russianstory** [comment]
![image](https://user-images.githubusercontent.com/48765277/58866317-027bf900-86c1-11e9-915e-7e51e12b2a7c.png)

**Comment by PodeCaradox** [comment]
https://github.com/Gaaammmler/Gm1KonverterCrossPlatform/releases
You can paste the programm wherever you want, maybe start it with administrator rights. Also you maybe deleted your workfolder? 
Did it stopped working when you click on a GM1 file or on starting the programm?

**Comment by Russianstory** [comment]
Running as Administrator did not help. Same. Working folder? may be. How to do it?

**Comment by PodeCaradox** [comment]
One idea is to delete the userconfig folder under C:/Users/your username/appdata/roaming here delete the GM1converterCrossPlatform folder.
the folder appdata is not visible you can just write %appdata% into windows search.

**Comment by Russianstory** [comment]
YES right! exactly! now everything is working! Thank you =)

**Comment by PodeCaradox** [comment]
I will add a Error message to the next release if that happens again. You need to select workfolder and Gm1 folder again, because of the delted userconfig. ;)

**Comment by Russianstory** [comment]
@Gaaammmler Your program is super! It works as reliably as AK-47)))))) thank you. It is worthy of respect! Especially on the background of the misery that the official developers of the game offered us. Now the question is: can you please make it so that when you click on the image of the object and in general, by default, the data on the image (object) is specified: height, length, depth, resolution, pixels. For example, I often get asked how many pixels do I need for a barn? And I don’t know ... and it would be great if all this would be taken into account in the program. Low bow. Thank you very much!

=

Твоя программа супер! Она работает так же надёжно как АК-47 )))))) спасибо тебе Это достойно уважения! Тем более на фоне того убожества, что предложили нам официальные разработчики игры. Теперь вопрос: а ты можешь пожалуйста сделать так, чтоб  и при нажатии на картинку объекта и вообще по-умолчанию прописывались данные о рисунке (объекте): высота, длина, глубина, разрешение, пиксели. Меня например очень часто спрашивают а сколько пикселей надо для амбара? А я и не знаю... а так было бы здорово если в программе это всё было бы учтено. Низкий поклон. Спасибо большое!

**Comment by PodeCaradox** [comment]
Im not exactly sure what you mean, can you specifiy what you want me to implement? or draw a picture which surface you want me to add? Dou you mean some fields that show you resolution of the Image you selected,(its width * height). what you mean with depth? I can add some values to the Images and datatype: 
For Images:
        private ushort width;
        private ushort height;
        private ushort offsetX;
        private ushort offsetY;
        private byte imagePart;
        private byte subParts;
        private ushort tileOffset;
        private byte direction;
        private byte horizontalOffsetOfImage;
        private byte buildingWidth;
        private byte animatedColor;
        private UInt32 offsetinByteArray;
        private UInt32 sizeinByteArray;

For Datatypes:
        private String name;
        private UInt32 iUnknown1;
        private UInt32 iUnknown2;
        private UInt32 iUnknown3;
        private UInt32 iNumberOfPictureinFile;
        private UInt32 iUnknown4;
        private UInt32 iDataType;
        private UInt32[] iUnknown5 = new UInt32[14];
        private UInt32 iDataSize;
        private UInt32 iUnknown6;

For example, I often get asked how many pixels do I need for a barn?
Ehm with which Imagetool you working? you can see the pixel size in mostly every Image programm. But i can implement that esaliy

**Comment by Russianstory** [comment]
![image](https://user-images.githubusercontent.com/48765277/58986624-7e239600-87e6-11e9-8a00-b6f04eb5b8d3.png)

**Comment by PodeCaradox** [comment]
For Building height the middle line or left/right side or both? and for the width top and bottom in your picture and the diagonal width there?

width need to be the same. For the height you can change it like you want but the toptiles have a fixed position inside stronghold if anyone find that in the exe you change that to in future. ;)

**Comment by Russianstory** [comment]
editing )))) 
![image](https://user-images.githubusercontent.com/48765277/58987407-1706e100-87e8-11e9-95da-00543338f4ab.png)

**Comment by PodeCaradox** [comment]
Do you have Discord,Skype or teamspeak @Russianstory ?

**Comment by Russianstory** [comment]
no sorry) i cannot speak well. i use translator)

**Comment by PodeCaradox** [comment]
Here some explinations: first the roofs have static positions inside Stronghold exe look here for my issue i need help: https://github.com/Sh0wdown/UnofficialCrusaderPatch/issues/375

also you not need the size from the middle tile in green, you can change the hole width of the Image but the witdh cant be changed from the .png. The picture or the Tiled Datatype consists like in the Info button shown of a Tiled Base and Images on top of it. Here examples:
(Under the tiled diamond base all will be not used, and all on top of it you can change like you want)
![Image13](https://user-images.githubusercontent.com/5760157/58988423-f8055080-87e1-11e9-804d-0980ce96f029.png)
![Image26](https://user-images.githubusercontent.com/5760157/58988424-f89de700-87e1-11e9-8613-7ce25aec596c.png)
![Image7](https://user-images.githubusercontent.com/5760157/58988425-f89de700-87e1-11e9-9b7c-6adfa57e708c.png)
![Image10](https://user-images.githubusercontent.com/5760157/58988426-f89de700-87e1-11e9-842e-a3a2a82e75dd.png)

Discord will be easier to write and anwering faster you need not to talk ;)


The static tiles on Top that dissapear if anyone stand on top of the Tower, i can them make invisible from start on, maybe i add this in the Programm:
![Image124](https://user-images.githubusercontent.com/5760157/58989335-048aa880-87e4-11e9-850d-4b67989d7d0d.png)
![Image122](https://user-images.githubusercontent.com/5760157/58989336-048aa880-87e4-11e9-8bbc-04d0c1022a38.png)
![Image123](https://user-images.githubusercontent.com/5760157/58989337-048aa880-87e4-11e9-8e8c-2e0b56ca12a4.png)

And the other static Images that have the Positions inside Stronghold exe:
![Image3](https://user-images.githubusercontent.com/5760157/58989388-1ec48680-87e4-11e9-8bcb-cf0cdfbf6ad3.png)
![Image4](https://user-images.githubusercontent.com/5760157/58989390-1ec48680-87e4-11e9-8edf-50b7db9fcec1.png)
![Image2](https://user-images.githubusercontent.com/5760157/58989391-1ec48680-87e4-11e9-9d99-6d314bbc33e6.png)
![Image24](https://user-images.githubusercontent.com/5760157/58989396-208e4a00-87e4-11e9-8048-5c23964dfd2b.png)

**Comment by PodeCaradox** [comment]
![Image10](https://user-images.githubusercontent.com/5760157/58988819-cd67c780-87e2-11e9-8e9c-ce6faf8577eb.png)

all on Top the red area can also be changed here

**Comment by Russianstory** [comment]
OK I understand you thank you so much for the information and picture) I will make myself an instruction to look for artists.

**Comment by Russianstory** [comment]
@PodeCaradox Hello my friend. Tell me please. What are the coordinates of the buildings in the game? How to determine this? how rotated the building is and aspect ratio. I need to know to work in the 3D program Blender. The fact is that you must first expose the coordinates of the buildings so that they are ideally fitted to the coordinates of the original buildings from the game.

=

Привет дружище. Скажи пожалста. Какие координаты зданийв игре? как определить это? насколько повёрнуто здание и соотношение сторон. Мне надо знать для работы в 3д программе Блендер. Дело в том, что надо сначала выставить коррдинаты зданий, чтоб они были иделально подогнаны под коррдинаты оригинальных зданий из игры.

**Comment by PodeCaradox** [comment]
@Russianstory the game is isometric so all is 2d but isometric mean it give you the look of 3d but the hole game is 2d. Here is a page on wiki that explain isometric a little bit more: https://en.wikipedia.org/wiki/Isometric_projection
What you want to do with blender? you want to change textrues from 3d model and save them as the 2d Images?

**Comment by Russianstory** [comment]
I will remodel all the buildings. The graphics will be redone in Russian style. I will do it in the 3D program and then take pictures of the 3D model in the angle I need. Therefore, I asked what coordinates or whether they exist at all to set the correct angle, tilt and general appearance of the building.

**Comment by PodeCaradox** [comment]
you can calulate them fom the images height and width with the angle, just convert it back from isometric view to get height and witdh for 3d model.
something like that shoul work:
realwidth = TileWidth * Cos(30) + TileHeight * Cos(60)
realHeight = TileWidth * Sin(30) + TileHeight * Sin(60)

**Comment by Russianstory** [comment]
@PodeCaradox Thank you very much. I understood. Please pozhalsta. extract all frames of trees and two bushes with animation from the game. I beg you

**Comment by PodeCaradox** [comment]
With the Gm1 exporter Gif exporter you can do this easily
![Unbenannt](https://user-images.githubusercontent.com/5760157/64046164-75372c00-cb6b-11e9-969b-15aa881a4e1c.JPG)
Just select all tiles with ctrl + a and click gif export.

![ImageAsGif](https://user-images.githubusercontent.com/5760157/64046224-ae6f9c00-cb6b-11e9-99c9-bdc1ff8ea3c1.gif)

**Comment by Russianstory** [comment]
ok i will try

**Comment by Russianstory** [comment]
@PodeCaradox  Why, when you export animation graphics, I get ten OrginalAnimationPalette folders in each of which 148 frames were .... is it the same thing? I can `t get it

**Comment by PodeCaradox** [comment]
@Russianstory do you donwloaded the new gm1 converter from the release page?
https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/releases
just click here and the gif exporter will shown
![Unbenannt](https://user-images.githubusercontent.com/5760157/64049345-cf88ba80-cb74-11e9-8bdd-503d7b221bab.JPG)

**Comment by Russianstory** [comment]
YEs i downloaded  converter from the release page

**Comment by PodeCaradox** [comment]
also the neweset version?

**Comment by Russianstory** [comment]
it is ok now i have animation )) trees and bush )

**Comment by Russianstory** [comment]
@PodeCaradox Hi, why can't I work in a program with files from the gfx folder? I really need pictures where in the barn there is food in different quantities .... preferably in full 250 units. How can I do this? Thanks =)

Привет, а почему я не могу работать в программе с файлами из папки gfx ? Мне очень нужны кадры где в амбар есть еда в разном количестве.... желательно в полный 250 единиц.Как мне это сделать? Спасибо =)

**Comment by PodeCaradox** [comment]
Oh i didn't add this folder by now, i can make fast a new release with the folder automatically in it.

**Comment by Russianstory** [comment]
I would really appreciate it. Do it please. I need to fit the barn to the original barn.

**Comment by PodeCaradox** [comment]
@Russianstory i added the gfx folder now just follow the instructions on the new release and the gfx Folder will show up: https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/releases

If you find a error just send me a message i didn't test the release so much for now.

**Comment by Russianstory** [comment]
I can’t understand how it works all the time, it gives an error .... it seems that everything was fine with the gm folder ... but it’s not with the gfx folder - it gives an error. And I have to erase everything in the C / name / roaming drive I do not understand ....

**Comment by PodeCaradox** [comment]
can you send me a picture with the error? maybe I know your problem, also can you show me your folder structure inside the stronghold folder. 

Is you gfx Folder inside the same folder as the gm?
![preview1](https://user-images.githubusercontent.com/5760157/65770494-8194b180-e136-11e9-8ef4-e8d30ee8f46a.JPG)

**Comment by Russianstory** [comment]
![image](https://user-images.githubusercontent.com/48765277/65777545-e22ee900-e14c-11e9-904f-31f278a8d24c.png)

![image](https://user-images.githubusercontent.com/48765277/65777626-10142d80-e14d-11e9-9053-da591458f9fe.png)

**Comment by PodeCaradox** [comment]
Ok can you use this version activate Logger with the Checkbox in the Options menu and than restart the Program and send me the Logfile here. You can find the Logfile in the options menue open the Log folder
So i will know the error you get in my language.
[Gm1Converter.zip](https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/files/3663194/Gm1Converter.zip)

**Comment by Russianstory** [comment]
thx i will try soon

**Comment by Russianstory** [comment]
Hey. Yes, the folder seems to work)) Thank you very much. Tell me, if I ask you to introduce new graphics for my mod to introduce to the game, will you help?) I will do it crookedly and you will be much more experienced in this matter.

**Comment by PodeCaradox** [comment]
yeah if he has any questions he can just ask me here. Do the converter now work no error anymore?

**Comment by Russianstory** [comment]
Sorry, I had a mistake in the text.
I will upload here only me. This is a good thing to do. I asked for your help. Replace the old house with a new one. Yes, the program seems to be working properly.

**Comment by PodeCaradox** [comment]
Also the gfx folder visible to you? Can you pls send me the log file? You can replace it with the editor or how you need my help exactly?

**Comment by Russianstory** [comment]
This one  log file
[LogFile from 2019.9.27.txt](https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/files/3665927/LogFile.from.2019.9.27.txt)

**Comment by PodeCaradox** [comment]
ah ok gfx folder work for you now. I see no error in the log file:)

ps: you can deactivate the logger ;)

**Comment by Russianstory** [comment]
um ... well, I can probably)) the truth is, everything seems to suit me) but what?)

**Comment by PodeCaradox** [comment]
you can now deactivate the logger in the options ;)

**Comment by PodeCaradox** [comment]
@Russianstory a new update will comming soo, i now can change the offsets in the exe so you can make bigger roofs from tower, gates, etc. :
![test](https://user-images.githubusercontent.com/5760157/67370182-89e7dd00-f57a-11e9-99d8-1954bdc51290.jpg)
![unknown](https://user-images.githubusercontent.com/5760157/67370183-89e7dd00-f57a-11e9-8ac2-bf1e8972dd17.jpg)

**Comment by Russianstory** [comment]
@PodeCaradox thanks for the news!))
Tell me, can I ask you to help me pack new files? (buildings) It's just that you do it better and I trust you more than myself) please?)))

We are slowly but in the process of creating graphics for modification. Very slowly because we do it from scratch and do it in size of the original.

**Comment by PodeCaradox** [comment]
@Russianstory hmm yea maybe i can help you, so i can explain you the compiler better that you can work without me on it ;)

**Comment by Russianstory** [comment]
Well, if you are uncomfortable or do not want to, I understand ok without problems))

**Comment by PodeCaradox** [comment]
No its OK for me, so I can improve the converter for easier use. I want to help so we can get togheter a better stronghold. I will help you to improve your skills with my converter.

**Comment by Russianstory** [comment]
@PodeCaradox Hi can u help me. Can u plz replace the original tower with this one.

**Comment by Russianstory** [comment]
![ROUND TOWER BIG](https://user-images.githubusercontent.com/48765277/68543313-9d8ea280-03c6-11ea-9da3-cef0dbbca883.jpg)

**Comment by Russianstory** [comment]
![Ja4mPFrOkIM](https://user-images.githubusercontent.com/48765277/68544294-bc933180-03d2-11ea-82b9-7da71da181be.jpg)

**Comment by PodeCaradox** [comment]
@Russianstory Is this the isometric view of the tower? And i need it as .png not .jpg (without white background, instead transparent background)

**Comment by Russianstory** [comment]
Ok we will try our best

**Comment by Russianstory** [comment]
[bashnya.zip](https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/files/3828863/bashnya.zip)

**Comment by Russianstory** [comment]
@PodeCaradox FRIEND PLEASE pack this tower instead of a big round one. I need to look at the size and angle, and is everything okay if you send units to the roof there. As you do, I ask you to drop me a ready GM1 file.

**Comment by PodeCaradox** [comment]
Here test it with that:
[test.zip](https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/files/3829053/test.zip)
![grafik](https://user-images.githubusercontent.com/5760157/68552179-69da6980-0414-11ea-91fd-41b04a2bfde6.png)

I did maybe the roof to high x)

**Comment by PodeCaradox** [comment]
Also can you maybe test this version of the converter, if it work on you Computer?
[Gm1KonverterCrossPlatform.zip](https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/files/3829120/Gm1KonverterCrossPlatform.zip)

Also for the next Release i plan the following to add: 
Pack all Files you modded to a Texture pack File and you can add Authors and Preview Text + Images.
I create than a second Simple Program where Users can Manager there Texture packs(Texture pack Files) and change them like they want. Do you want me to include some info's more or will that be fine?

**Comment by Russianstory** [comment]
Excellent))) it turns out that the tower is made correctly, as required by the game? SO?

**Comment by Russianstory** [comment]
About the roof did not quite understand you. What is meant?)

**Comment by Russianstory** [comment]
Tell me about the tower - is everything okay with it? can it be added to the game? What are the problems and jambs on the tower. Say everything I could see)

**Comment by PodeCaradox** [comment]
It looks like all should be fine but you need some stones on the start from the Tower, your Tower is a little bit Smaller than the previous one.

**Comment by Russianstory** [comment]
Tell me, and even better, graphically show how much less? If I may.  ))

**Comment by PodeCaradox** [comment]
I did post it above the new GM1 Files and a Image of your Tower ingame, just download the test.zip. I need to change the height so it looks right. But you need to fill the bottom for the Rectangle there. 
https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/issues/2#issuecomment-552235727

**Comment by Russianstory** [comment]
THAT is to look right ?? everything seems to be the same. Roof?? There was no roof at the largest tower

**Comment by Russianstory** [comment]
The towers are the same in height. Checked. All right. There is a roof / floor. The stone laying at our towers is different than in these, so the base of the tower is just that.

**Comment by PodeCaradox** [comment]
I mean the height of the Image that is before the Units. Yeah i cant draw transparent at the bottom it needs to be filled with something or i make a sand texture from the Original Stronghol under it.

Here you can see it is filled with that Stones that it build a Rectangle:
![grafik](https://user-images.githubusercontent.com/5760157/68608531-7530a200-04b3-11ea-9b5f-3856bac68456.png)

Also can we use Discord for easier communication?
https://discord.gg/XHThHc

**Comment by Russianstory** [comment]
I did not understand about height for units again. The height of the teeth which are in a circle around the tower? Or is it an incorrect translation of an online translator.

**Comment by Russianstory** [comment]
I understood about the bottom of the building. We have no such basis. I took the image of the original Russian tower. There are no piled stones below. There simply goes masonry of stones.

**Comment by Russianstory** [comment]
![КРУГЛАЯ БАШНЯ](https://user-images.githubusercontent.com/48765277/68608710-9b127280-04c4-11ea-990f-6cec3894fd3f.png)
this one

**Comment by PodeCaradox** [comment]
i meant that i need to change the height that it exact match the right height, i did it not perfect, so thats fine.

![Image3](https://user-images.githubusercontent.com/5760157/68608688-ca6cb380-04b3-11ea-8e51-091fedc8d25c.png)
For the Bottom i can place the Standard Dessert Texture that it works.

Do you have Discord? So can i reply faster in future.

Also great work :)

**Comment by Russianstory** [comment]
Yes there is a discord)) but there I also write through a translator.
And that means about the height of the walls themselves on the roof, About the height of the walls in front of which swordsmen are standing. So? But we have already made a tower)). The only thing is that it will be possible to expand the battlements on the tower like the original.
And apply the texture now? Just curious to see.
We will apply our texture to the mod., Then I will throw the finished file with the texture here.

**Comment by PodeCaradox** [comment]
thats ok just join the discord its easier for me so i check it more often than github.
I will add you the files one mom, i also found a bug in my Gm1Editor xD I dont need change exe values with that bug lol.

**Comment by PodeCaradox** [comment]
here just replace the gm1 files
[test.zip](https://github.com/PodeCaradox/Gm1KonverterCrossPlatform/files/3832632/test.zip)

just comme discord so i can write your there

**Comment by PodeCaradox** [comment]
Or give me your Discord Name so i can add you

**Comment by Russianstory** [comment]
Roland#0471

**Comment by Russianstory** [comment]
Hi - when I try to click the image import, my program crashes with an error.

**Comment by alexeik** [comment]
@Russianstory
привет.
слушай, я могу помочь перевести игру на движок openRA.

**Comment by iamWitchKing** [comment]
Hello buddy , I am modder , I built some mod and idea but I am vacant hand I can not working as quick , I need crew , if you interested in see my project,  please contact me soon as possible. 

Twitter : @iamWitchKing

Telegram: @iamWitchKing
